src/Money.php

```diff
@@ -76,6 +77,7 @@ public function __construct($amount, Currency $currency)
 
         $this->amount = (string) $amount;
         $this->currency = $currency;
+        self::$calculator = $calculator;
```

Comment:
This wouldn't be a good idea anyway, because it sets a global instance.

Reply:
Thanks for the remark. If my memory serves me, I took that line of code from the following method:
```
     /**
      * @return Calculator
      */
     private function getCalculator()
     {
         if (null === self::$calculator) {
             **self::$calculator = self::initializeCalculator();**
         }
 
         return self::$calculator;
     }
```

Reply:
Yep, `disableArbitraryPrecisionCalculators` is really helpful in my opinion. Anyway I still wonder if it makes any sense to suggest that `BcMathCalculator` and `GmpCalculator` can be moved from `moneyphp/money` to a new repo (`moneyphp/big-money`).